modules/nf-core/ilastik/multicut/main.nf

```diff
@@ -0,0 +1,38 @@
+process ILASTIK_MULTICUT {
+    tag "$meta.id"
+    label 'process_low'
+
+    container "labsyspharm/mcmicro-ilastik:1.6.0"
+
+    input:
+    tuple val(meta), path(h5)
+    path ilp
+    path probs
+
+    output:
+    tuple val(meta), path("*.tiff") , emit: out_tiff
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    /ilastik-release/run_ilastik.sh \\
+        --headless \\
+        --readonly 1 \\
+        --project=$ilp \\
+        --raw_data=$h5 \\
+        --probabilities=$probs \\
+        --export_source="Multicut Segmentation" \\
+        $args
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        ilastik: \$(/ilastik-release/run_ilastik.sh --headless --version)
+    END_VERSIONS
+    """
+}
```

Comment:
Can you also add the stub?

Reply:
You can see an example here: https://github.com/nf-core/modules/blob/9c8154a8dd4989ddcbd540f78ffcd98c04aec767/modules/nf-core/vcf2db/main.nf#L38-L49

I agree this could be better documented, but it's basically adding a little script that should be run when `-stub` is used in the nextflow command. It should create empty files that correspond what the module would output if it were fully run.

Reply:
I added a stub to the file, is this the way it should look?

https://github.com/FloWuenne/modules/blob/3e58ed1c129c372e3e88c79046affa31e132b46b/modules/nf-core/ilastik/multicut/main.nf#L40-L50